doc/_static/extract_api_implementation.py

```diff
@@ -0,0 +1,100 @@
+import base64
+import time
+import sys
+import json
+import requests
+
+account_token = "integration_token"  # Use your token
+domain_name = "https://iap-extract.odoo.com"
+path_to_pdf = "/path/to/invoice.pdf"
```

Comment:
It should be something more generic than PDFs as we also handle few image formats.
Something like `path_to_invoice_file` and `/path/to/invoice_file` (without explicit extension).